tests/dummy/app/pods/view/renderers/documentation/checkbox-array.md

```diff
@@ -29,3 +29,36 @@ Change what size checkboxes are used. See [ember-frost-core](http://ciena-frost.
   }
 }
 ```
+
+#### renderer.data
+
+Overrides the enum in the model to use specified label and value.
+
+```json
+{
+  "model": "foo",
+  "renderer": {
+    "name": "checkbox-array",
+    "data": [
+      {
+        "label": "The label for the checkbox",
+        "value": "The value for the checkbox"
+      }
+    ]
+  }
+}
+```
+
+#### renderer.selectedValues
+
+Allows checkboxes to be pre-checked. E.g. for retaining checks from query params upon refresh.
+
+```json
+{
+  "model": "foo",
+  "renderer": {
+    "name": "checkbox-array",
+    "selectedValues": ["values", "of", "selected", "checkboxes"]
```

Comment:
I don't think you need a separate `selectedValues` in the view, can't this be driven by the current value of that part of the model? Or by `default` specified in the model itself?